README.md

```diff
@@ -9,6 +9,15 @@ James is build with [hoxy](https://github.com/greim/hoxy), [electron](https://gi
 
 ![](resource/screenshot-1.png)
 
+## Installing
+Download the correct version for your OS and run
+
+![](https://img.shields.io/badge/download-OSX-green.svg?link=https://github.com/james-proxy/james/releases/download/v1.3.2/James-darwin-x64.zip)
+![](https://img.shields.io/badge/download-linux--ia32-green.svg?link=https://github.com/james-proxy/james/releases/download/v1.3.2/James-linux-ia32.zip)
+![](https://img.shields.io/badge/download-linux--x64-green.svg?link=https://github.com/james-proxy/james/releases/download/v1.3.2/James-linux-x64.zip)
+![](https://img.shields.io/badge/download-windows--ia32-green.svg?https://github.com/james-proxy/james/releases/download/v1.3.2/James-win32-ia32.zip)
+![](https://img.shields.io/badge/download-windows--x64-green.svg?https://github.com/james-proxy/james/releases/download/v1.3.2/James-win32-x64.zip)
```

Comment:
Each shield should link to each file, right now clicking on a shield simply opens the image in a new tab :)